.travis.yml

```diff
@@ -38,6 +39,7 @@ script:
   - make -j$(sysctl -n hw.ncpu)
   - make feature-test
   - make feature-test-clean
+  - mvn apache-rat:check
```

Comment:
what about move rat check before make hawq. since if rat check fail, no need to build at all.